styles/modules/modals/_editListing.scss

```diff
@@ -7,6 +7,36 @@
       min-height: 150px;
     }
 
+    .tile {
+      width: 100px;
+      height: 100px;
+      border: 1px solid;
+      border-color: inherit;
+      border-radius: $corner;
+      float: left;
+      margin-right: $pad;
+      margin-bottom: $pad;
+      position: relative;
+
+      .closeIcon {
+        width: 24px;
+        height: 24px;
+        position: absolute;
+        top: 0;
+        left: 0;
+      }      
+
+      button {
```

Comment:
Yes, will do. Also, as an FYI, the parent of this guy is .tile which is something I'll re-use when I end up coding up attachments:

![image](https://cloud.githubusercontent.com/assets/794517/19614148/c0baba12-97a6-11e6-8474-2ead02d41d07.png)

That's something you should be able to re-use for the listing detail view (without the close icon). I'll probably make a template for it and we could put that template and the tile class in a location less specific to the edit modal.

But, i think we could hold off on doing that until the server actually implements attachments.